core/domain/feedback_services.py

```diff
@@ -167,7 +167,7 @@ def create_message(
     msg.put()
 
     # Update the message count in the thread.
-    if thread.message_count:
+    if thread.message_count is not None:
```

Comment:
@nithusha21, After going through the history I found these changes were made[ here](https://github.com/oppia/oppia/pull/6978/files#diff-e8e75b25b618133593df5ff9cd67a59bR170).

Reply:
Hmmm, @lilithxxx could you please take a look at this?

Reply:
Hm I thought message_count is an integer and that's why I removed the None check. Reverting this change will decrease the coverage of the code_base though. [This](https://codecov.io/gh/oppia/oppia/compare/f8d99190f294e8c3d93fc855c5b89b88c583b264...f803319552338b3c3c007fddbe112f21ed6f9cd3/src/core/domain/feedback_services.py?before=core/domain/feedback_services.py#L173) line will get untested.

Reply:
I meant this line:
https://github.com/oppia/oppia/blob/50e8ae3171578d335df5783f4b8910af840ce1c2/core/domain/feedback_services.py#L173

When exactly can message_count be None?